modules/orionode/index.js

```diff
@@ -19,60 +18,201 @@ var express = require('express'),
 	checkRights = require('./lib/accessRights').checkRights,
 	socketio = require('socket.io'),
 	ttyShell = require('./lib/tty_shell'),
-	responseTime = require('response-time');
+	responseTime = require('response-time'),
+	//lsregistry = require("./lib/lsRegistry"),
```

Comment:
nit: Delete commented code?

Reply:
It is only commented out so I can release the module loading code - it is used in the forthcoming language server feature